### ISSUE TITLE
[Feature request] Add option to auto send mission that are not elite/rare ignoring selected categories.

### ISSUE THREAD

**Issue by czullo** [teelolws/TLDRMissions#197]
**TLDR Version of your idea**
TLDR version: Add option to auto send mission that are not elite/rare

95% of non elite/rare missions are 10 anima cost, and they are waste of space if you need to wait them to expire and they block "spot" por better missions like gold. So if you could bring option to automaticaly calculate all non elite/rare mission even if they dont match selected categories. its because for example i want to do pet charms that are regular missions for 10 anima but dont want do pet charms that are 50 anima. but if i reduce mission cost with slider to for example 10 anima i will miss missions with xp follower tokens and anima missions that cost 50 anima and above.

**Comment by teelolws** [comment]
Being able to specify anima limits per category is something on my list! https://github.com/teelolws/TLDRMissions/issues/107
